tests/core/creators/_helpers/tools.js

```diff
@@ -5,6 +5,8 @@
 var detachedTests = ( function() {
```

Comment:
Most tests that check against editor not being null are now false positive, as a function callback is also not null. Also, we are not ensuring that the delayed initialization feature has been cleaned up which impacts test isolation.